microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/WebXmlActivationCompatibilityTest.java

```diff
@@ -0,0 +1,72 @@
+package org.jboss.eap.qe.microprofile.jwt.activation;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.jwt.auth.tool.JsonWebToken;
+import org.jboss.eap.qe.microprofile.jwt.auth.tool.JwtHelper;
+import org.jboss.eap.qe.microprofile.jwt.auth.tool.RsaKeyTool;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.NonJwtJaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.restassured.RestAssured;
+
+@RunWith(Arquillian.class)
+public class WebXmlActivationCompatibilityTest {
```

Comment:
See my previous comment about the class Javadoc.

Reply:
javaDoc added, thanks.